install.ps1

```diff
@@ -0,0 +1,307 @@
+#Requires -Version 5.0
```

Comment:
do we really want a windows install script? can we not rely exclusively on pkg managers? 

I don't know powershell well enough to review or maintain this, and i don't know if windows users actually install shit using this method -- in my head, it's more of a linux thing to deal with the fact that there are so many different package ecosystems.